docs/Manual/03 Schedules.md

```diff
@@ -2,7 +2,7 @@
 [//]: # (Version: v1.2.1)
 
 # Section 3: Schedules
-We begin this section with `nest` from [Section 2](<02%20Simple%20Affine%20Loop%20Nests.md>), which captures the logic of matrix-matrix multiplication. We use `nest` to create a `Schedule`, which controls the execution order of the nest iterations. Schedules are target-independent, in the sense that the same schedule can be used to emit code for multiple target platforms.
+This section begins with `nest` from [Section 2](<02%20Simple%20Affine%20Loop%20Nests.md>) that captures the logic of matrix-matrix multiplication. `Nest` is used to create a `Schedule` that controls the execution order of the nested iterations. Schedules are target-independent because the same schedule can be used to emit code for multiple target platforms. 
```

Comment:
"nest iterations" is more accurate because iterations is a bit ambiguous (for example, iterations can mean how index i goes from 0 to 10, step 1, or iterations can generally refer to all the iterations of an index).

Or perhaps we should say "the execution order of the nest's iterations."

Reply:
"the execution order of the nest's iterations." sounds more accurate